src/views/officer_tacoma.j2

```diff
@@ -3,4 +3,4 @@
 <p><b>Title*:</b> {{ title }}</p>
 <p><b>Department*:</b> {{ department }}</p>
 <p><b>Salary*:</b> {{ salary }}</p>
-<p><i>*data is from 2019</i></p>
\ No newline at end of file
+<p><i>*data is from {{ date }}</i></p>
```

Comment:
Can we make this message the same format as the Seattle one for consistency?

Reply:
Whoops, yep.